api-spec/protobuf/tdextrade/v2/swap.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package tdextrade.v2;
```

Comment:
Why not just tdexswap? also there were no v1 of this field before?

```suggestion
package tdexswap;
```

Reply:
Done